core/src/test/java/brooklyn/location/basic/SshMachineLocationIntegrationTest.java

```diff
@@ -63,7 +63,14 @@ public void tearDown() throws Exception {
         mgmt = null;
     }
 
-    // Note: requires `named:localhost-passphrase` set up with a key whose passphrase is "localhost"    
+    // Note: requires `named:localhost-passphrase` set up with a key whose passphrase is "localhost"
+    // * create the key with:
```

Comment:
Excellent! We need more of this kind of thing, and to get in the habit of writing it at the same time as the test (me included!).